api/src/identity-access-management/infrastructure/repositories/last-user-application-connections.repository.js

```diff
@@ -0,0 +1,13 @@
+import { knex } from '../../../../db/knex-database-connection.js';
+
+async function save({ userId, application, lastLoggedAt }) {
```

Comment:
Ne serait-il pas plus explicite de nommer cette fonction `upsert` indiquant ainsi clairement que si ça existe c'est mis à jour et sinon c'est créé ?

```suggestion
async function upsert({ userId, application, lastLoggedAt }) {
```

Reply:
Tout à fait 👍